package.json

```diff
@@ -18,13 +18,14 @@
     "remixd": "remixd -s . --remix-ide https://remix.ethereum.org",
     "generate-types": "npm run generate-types:ethers",
     "generate-types:ethers": "npx typechain \"build/contracts/*.json\" --target=ethers-v5 --outDir src/ethers",
-    "deploy-local": "truffle migrate && truffle exec scripts/create_markets.js --network ${NETWORK:-development}",
+    "deploy-local": "truffle migrate && truffle exec scripts/markets/create_markets.js --network ${NETWORK:-development}",
     "deploy-kovan": "NETWORK=kovan npm run deploy-local",
     "deploy-rinkeby": "NETWORK=rinkeby npm run deploy-local",
-    "deploy-mainnet": "truffle migrate && truffle exec scripts/mainnet_create_markets.js --network mainnet",
-    "transfer-ownership-local": "truffle exec scripts/transfer_ownership.js --network ${NETWORK:-development}",
-    "transfer-ownership-rinkeby": "NETWORK=rinkeby npm run transfer-ownership-local",
-    "transfer-ownership-mainnet": "NETWORK=mainnet npm run transfer-ownership-local",
+    "deploy-mainnet": "truffle migrate && truffle exec scripts/markets/mainnet_create_markets.js --network mainnet",
```

Comment:
the `scripts/` directory was getting crowded so I moved related scripts into their own subdirectories.